core/meta/src/TClass.cxx

```diff
@@ -4448,46 +4448,49 @@ TVirtualStreamerInfo* TClass::GetStreamerInfo(Int_t version /* = 0 */) const
 
 TVirtualStreamerInfo* TClass::GetStreamerInfoAbstractEmulated(Int_t version /* = 0 */) const
 {
-   R__LOCKGUARD(gInterpreterMutex);
+   TVirtualStreamerInfo* sinfo = nullptr;
 
-   TString newname( GetName() );
+   TString newname(GetName());
    newname += "@@emulated";
 
-   TClass *emulated = TClass::GetClass(newname);
+   R__LOCKGUARD(gInterpreterMutex);
 
-   TVirtualStreamerInfo* sinfo = 0;
+   TClass *emulated = TClass::GetClass(newname);
 
-   if (emulated) {
+   if (emulated)
```

Comment:
No, I just find it cleaner to not have braces on if's with a single statement. Using less braces means that we can fit more code in a screenful of text, so it's easier to read the code too. However, `clang-format` does want to always put these if statements on a single line, which annoys me, as the indentation helps when reading.